openapi/templates/rust/reqwest/api.mustache

```diff
@@ -8,6 +8,8 @@ use reqwest;
 use crate::apis::ResponseContent;
 use super::{Error, configuration};
 
+// temporarely needed until https://github.com/OpenAPITools/openapi-generator/issues/5676 is fixed
+use crate::models::CourierMessageStatus;
```

Comment:
I think this will break SDKs for hydra and keto which don't have this class.

Reply:
Yup, of course it does... Damn. 

Not sure how to proceed here, maybe disable rust for now?

Reply:
Found a more permanent solution that works. The diff is really hard to read, because the line is too long, I think. I'll try to explain what's going on, and can also show it to you. :)

There is another property in the parameters of the operation called `schema`. That contains the `dataType` prefixed with `crate::models::` as well. So I use that, if it's available. If not, we fall back to the old way. That should work and everything is compiling, so in the spirit of Rust's "if it compiles, it works" mantra, I am confident that this produces the correct result.